peerinst/templates/peerinst/collection/featured_collection_list.html

```diff
@@ -0,0 +1,149 @@
+{% extends 'peerinst/base.html' %}
+
+{% load bleach_html staticfiles compress i18n %}
+
+{% block stylesheets %}
+{{ block.super }}
+{% compress css %}
+<link href="{% static 'peerinst/css/collection.min.css' %}" rel="stylesheet">
```

Comment:
If you're repeating a lot of code in all your `collection` html files, it would be worth it to have them extend a common `peerinst/collection/base.html` file.